packages/toolpad-core/src/types.ts

```diff
@@ -306,6 +306,7 @@ export interface SerializedError {
   message: string;
   name: string;
   stack?: string;
+  code?: unknown;
```

Comment:
So it's for https://nodejs.org/api/errors.html#errorcode?

Reply:
It's a custom type, so perhaps we could rename it to better represent this (`SerializedToolpadError` ?)

Reply:
We standardize on regular JavaScript `Error`s with a `code` property similar to how node.js treats them. we frequently pass errors between server and client and this format is the most straightforward to do that with.

in general we should avoid using exceptions for control flow. that paradigm doesn't mesh well with a distributed system anyway.